### PR TITLE
Zoe-mappings changes

### PR DESCRIPTION
**Pull request by yourgirlcodes**

### CODE REVIEW

File: entities/index.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export * from './Audience';
 export * from './Audience/AudienceCondition';
 export * from './Journey';
 export * from './Journey/JourneyCondition';
+export * from './Application';
```
Comment (iBaryo) [inline entities/index.ts:19]: already in the export list...

File: entities/Audience/AudienceCondition.ts [hunk 2]
```diff
@@ -11,83 +11,83 @@ export type AudienceCondition = Condition<
...
-                operand: {
-                    type: 'double',
-                    value: 1000
-                }
-            }
-        }
-    ]
-};
\ No newline at end of file
+// const conditionExample: AudienceCondition = {
```
Comment (iBaryo) [inline entities/Audience/AudienceCondition.ts]: comment out...

File: entities/Application/index.ts [hunk 3]
```diff
@@ -38,6 +36,7 @@ export interface ApplicationAuth {
 export interface ApplicationsEndpoints {
     '/businessUnit/$bUnit/applications': {
         GET: (bUnit: Id) => Promise<Application[]>;
+        //TODO: fix this payload...
```
Comment (iBaryo) [inline entities/Application/index.ts]: ?

File: CDPEntitiesApi.ts [hunk 4]
```diff
@@ -46,15 +39,15 @@ export type CDPEntitiesApi = {
         }>;
     }>,
     businessunits: EntityApi<CDPEntityDef<BusinessUnit>, {
-        mappings: EntityApi<CDPEntityDef<Record<string, Array<{ sourceField: string; targetField: string }>>>>; // deprecate this
+        // mappings: EntityApi<CDPEntityDef<{ sourceId: string; targetId: string; mappings: Array<{ sourceField: string; targetField: string }> }>>; // deprecate this
```
Comment (iBaryo) [inline CDPEntitiesApi.ts]: remove

File: CDP.ts [hunk 5]
```diff
@@ -37,8 +36,7 @@ export class CDP {
         proxy: undefined as string,
         ignoreCertError: false,
         verboseLog: false,
-        anonymousPaths: [] as RegExp[],
-        sendRequest: sendRequest
```
Comment (iBaryo) [inline CDP.ts]: revert all changes in this file..

**Review by iBaryo — CHANGES_REQUESTED** [review]

File: CDP.ts (hunk 5 above)
Comment (yourgirlcodes) [reply CDP.ts]: these changes we made together for sendRequest (applicable to e2e) - this was the 'create sendRequest abstract for SDK' step. If we remove sendRequest, I don't see how the integration with e2e will work..

Comment (yourgirlcodes) [reply CDP.ts]: we did these together for the sendRequest changes when we started e2e, if we remove sendRequest then I don't see how the sdk integration into the e2e will work.

**Comment by yourgirlcodes** [comment]
closed - see new

**Pull request closed without merging**